ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/arp.schema.yml

```diff
@@ -9,6 +9,20 @@ keys:
   arp:
     type: dict
     keys:
+      persistent:
```

Comment:
are we not adding `arp cache persistent` command support in this PR? if not then need to update PR title and description